packages/ckeditor5-select-all/src/selectallcommand.js

```diff
@@ -30,10 +31,20 @@ export default class SelectAllCommand extends Command {
 	 */
 	execute() {
 		const model = this.editor.model;
-		const limitElement = model.schema.getLimitElement( model.document.selection );
+		const selection = model.document.selection;
+		let limitElement = model.schema.getLimitElement( selection );
+		let place = 'in';
+		// If entire element was already selected, try selecting all in a parent limit element (if any).
+		if ( selection.containsEntireContent( limitElement ) && limitElement.root !== limitElement ) {
```

Comment:
Blank lines before `if()` and after `while()` are missing.

We need ESLint rule for that :D